packages/sveltekit/test/client/sdk.test.ts

```diff
@@ -47,5 +48,79 @@ describe('Sentry client SDK', () => {
       // @ts-ignore need access to protected _tags attribute
       expect(currentScope._tags).toEqual({ runtime: 'browser' });
     });
+
+    describe('automatically added integrations', () => {
+      it.each([
+        ['tracesSampleRate', { tracesSampleRate: 0 }],
+        ['tracesSampler', { tracesSampler: () => 1.0 }],
+        ['enableTracing', { enableTracing: true }],
+      ])('adds the BrowserTracing integration if tracing is enabled via %s', (_, tracingOptions) => {
+        init({
+          dsn: 'https://public@dsn.ingest.sentry.io/1337',
+          ...tracingOptions,
+        });
+
+        const integrationsToInit = svelteInit.mock.calls[0][0].integrations;
+        const browserTracing = (getCurrentHub().getClient() as BrowserClient)?.getIntegrationById('BrowserTracing');
+
+        expect(integrationsToInit).toContainEqual(expect.objectContaining({ name: 'BrowserTracing' }));
+        expect(browserTracing).toBeDefined();
+      });
+
+      it('merges the BrowserTracing integration with the user-provided one', () => {});
```

Comment:
l: should we also skip this test? And add a comment?

Reply:
Whoops, this is a duplicate of the test below, will remove it